### PR TITLE
Added more tables and fake data

### PR DESCRIPTION
**Pull request by wongband**
- All tables are rendered properly with fake data

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -26,25 +26,58 @@ function App() {
     const yearlyHeaders = [];
 
     for (let i = 0; i < 30; i++) {
-      yearlyHeaders.push({ name: 'Y' + (new Date().getFullYear() + i)});
+      yearlyHeaders.push({ name: "Y" + (new Date().getFullYear() + i) });
     }
 
+    // This part increases the width of a chosen column
+    const nameCol = worksheet.getColumn('B');
+    nameCol.width = 38;
```
Comment (srkirkland) [inline src/App.tsx:34]: is this used anywhere?

**Review by srkirkland — APPROVED** [review]
didn't run it but this looks good

File: src/App.tsx (hunk 1 above)
Comment (wongband) [reply src/App.tsx:34]: That line is used to make the column B in the excel sheet bigger. Thought it would make it look nicer

Comment (srkirkland) [reply src/App.tsx:34]: ah, that makes sense.  And I think overall it looks very nice, feel free to merge when you are happy with it 🚢

**Pull request merged**